packages/api/app/Utils/slugify.js

```diff
@@ -0,0 +1,38 @@
+const slugify = require('slugify');
+
+const incrementSlugSuffix = (oldSlug) => {
+	const slugSplitted = oldSlug.split('-');
+	const lastElementIndex = slugSplitted.length - 1;
+	// eslint-disable-next-line no-restricted-globals
+	const isLastElementNumber = !isNaN(slugSplitted[lastElementIndex]);
```

Comment:
Infelizmente o `Number.isNaN()` não funciona para esse caso, pois o que necessito aqui é apenas saber se a última parte da slug é "parseable to a number" ou não

Reply:
Assim dá certo: `const isLastElementNumber = Number.isInteger(Number(slugSplitted[lastElementIndex]));`

Ou assim:

`const isLastElementNumber = !Number.isNaN(Number(slugSplitted[lastElementIndex]));`

Se fosse com o `parseInt` não funcionaria muito bem, mas com o Number dá certo.
```javascript
Number('12px') === NaN
parseInt('12px') === parseInt('12') === 12
```

Reply:
Confesso que tentei apenas `Number.isNaN('xxx')` e não funcionou, vou tentar dessa maneira